ATTRIBUTION.md

```diff
@@ -590,6 +590,38 @@ The data are used to fit model for temperature-dependent fossil fuel usage.
 ##### General Purpose
 The data are used to fit model for temperature-dependent fossil fuel usage.
 
+##### Source
+* Name: Fraction Target for Reisidential and Commercial heat pump possession as main heating device
```

Comment:
We usually capitalize first letters for each word in `Name`, i.e. `Fraction Target for Residential and Commercial Heat Pump Possession as Main Heating Device`

Also, `Reisidential` -> `Residential`

Reply:
Got it